__tests__/app.test.js

```diff
@@ -166,3 +166,107 @@ describe("GET /api/reviews/:review_id/comments", () => {
 			});
 	});
 });
+
+describe("PATCH /api/reviews/:review_id", () => {
+	test("status:200, returns the updated review object when told to increment", () => {
+		const newVotes = {
+			inc_votes: 3
+		};
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(newVotes)
+			.expect(200)
+			.then(({ body }) => {
+				const { review } = body;
+				expect(review).toMatchObject({
+					owner: "mallionaire",
+					title: "Agricola",
+					designer: "Uwe Rosenberg",
+					review_id: 1,
+					category: "euro game",
+					review_img_url: "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+					created_at: "2021-01-18T10:00:20.514Z",
+					votes: 4
+				});
+			});
+	});
+
+	test("status:200, returns the updated review object when told to decrement", () => {
+		const newVotes = {
+			inc_votes: -1
+		};
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(newVotes)
+			.expect(200)
+			.then(({ body }) => {
+				const { review } = body;
+				expect(review).toMatchObject({
+					owner: "mallionaire",
+					title: "Agricola",
+					designer: "Uwe Rosenberg",
+					review_id: 1,
+					category: "euro game",
+					review_img_url: "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+					created_at: "2021-01-18T10:00:20.514Z",
+					votes: 0
+				});
+			});
+	});
+
+	test("status:204, returns nothing when not told to increment or decrement", () => {
+		const newVotes = {
+			inc_votes: 0
+		};
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(newVotes)
+			.expect(204)

```

Comment:
if you are returning a 204 there should be no content anyway that being said, I wonder why you have opted to send back a 204 rather than a 200, is it different to try and increase the votes by 0 that if I try to increase them by 1?